tests/test_all.py

```diff
@@ -43,6 +43,134 @@
 from nertokenizer import *  # noqa
 from settings import *  # noqa
 
+def checking(text, real):
+    resp = check_grammar(text)
+    i = list([ i["i"] for i in resp[0][0][0]['tokens'] ])
+    assert real == i
+    #if real != i:
+    #    print(text)
+    #    print(f"Result: {i}")
+    #    print(f"Real:   {real}")
+    #    pprint(resp)
+    #else:
+    #    pprint(resp)
+
+
+def test_character_spans():
+
+    # Person token tests
+    text = "Á Clinton."
+    real = [0, 1, 8]
```

Comment:
Is this correct? Shouldn't the array be [0, 1, 10]? (The 10 being the index of the period.)

Reply:
I've counted it so:

```
Á Clinton.
0 12345678

```
The period gets the index 8 here, according to the origin_spans count, that don't include the whitespace as such. 
If I change the code back to tracking t.original, I get [0, 1, 9], which seems to include the whitespace. 
I'll change it back and update the tests accordingly.

Reply:
Update done